include/6D9D80.h

```diff
@@ -5,14 +5,14 @@
 
 // void func_800B60E0_jp();
 // void func_800B6170_jp();
-// void mPB_keep_item();
+void mPB_keep_item(mActor_name_t name);
```

Comment:
It's the internal IDs for each actor, they're unique and each variation of actor have an ID

Reply:
I saw it defined in m_actor.h afterwards.
I'm not sure how I feel about it. In one hand it makes super clear the purpose of a parameter, it is an actor's name and not just a random number. In the other hand it forces every header that takes an actorname to include the `m_actor.h` header, which we want to try to avoid whenever possible. If header dependencies weren't as bad as they are then this would be completely fine to me

Reply:
That's true, so it can be kinda redundant.
And this codebase being a zelda-based one it would surprise me if from time to time we would need either a `s32`, `u8`, etc as the type to match a function, making this type less relevant.
So I think it would be better if it were removed. (Sad, I kinda liked the idea, but it is what it is)